frontend/src/lang/en.ts

```diff
@@ -568,9 +568,9 @@ export const en = {
         dateOfBirth: "Date of birth",
         enterDOB: "Enter your date of birth",
         enterDOB2:
-          "Enter your date of birth to access your COVID-19 testing portal.",
+          "Enter <0>{{personName}}</0>'s date of birth to access your COVID-19 testing portal.",
```

Comment:
@Rebecca-LM a couple of things here:

1) This sentence is now mixing second- and third-person, which I imagine we'll want to change: "[...] to access _their_/_this_/_the_ COVID-19 testing portal"?
2) We'll also require a minor update to the Spanish-language version of this message, which you can see here:
https://github.com/CDCgov/prime-simplereport/blob/nac/3184-add-patient-name-to-dob-validation-screen/frontend/src/lang/es.ts#L489-L490